lib/test_prof/factory_prof/printers/simple.rb

```diff
@@ -28,17 +28,54 @@ def dump(result, start_time:)
                Total time: #{total_time.duration} (out of #{total_run_time.duration})
                Total uniq factories: #{total_uniq_factories}
 
-                 total   top-level     total time      time per call      top-level time               name
+                 name                    total   top-level     total time      time per call      top-level time
             MSG
 
           result.stats.each do |stat|
-            time_per_call = stat[:total_time] / stat[:total_count]
-
-            msgs << format("%8d %11d %13.4fs %17.4fs %18.4fs %18s", stat[:total_count], stat[:top_level_count], stat[:total_time], time_per_call, stat[:top_level_time], stat[:name])
+            msgs << format("%-3s%-20s %8d %11d %13.4fs %17.4fs %18.4fs", *format_args(stat))
+            compressed_variations = compress_variations(stat[:variations], variations_limit)
+            compressed_variations.each do |variation_stat|
+              msgs << format("%-5s%-18s %8d %11d %13.4fs %17.4fs %18.4fs", *format_args(variation_stat))
+            end
           end
 
           log :info, msgs.join("\n")
         end
+
+        private
+
+        def compress_variations(variations, variations_limit)
+          limited_variation = {
```

Comment:
Let's rename it to `other_variation`

Reply:
I moved this logic to the #track method https://github.com/test-prof/test-prof/pull/296/files#diff-5bdf54a38f0e448125dcf0c2342a75e4c65f46ebb5a85f5ec8510dbba1b55f9eR139